stellargraph/connector/neo4j/graph.py

```diff
@@ -0,0 +1,113 @@
+__all__ = ["Neo4jStellarGraph"]
+
+import numpy as np
+import scipy.sparse as sps
+from ...core.experimental import experimental
+
+
+@experimental(reason="the class is not fully tested and lacks documentation")
+class Neo4jStellarGraph:
+    def __init__(self, graph_db, is_directed=False):
+
+        self.graph_db = graph_db
+        self._is_directed = is_directed
+
+    def nodes(self):
+        # FIXME: don't assume stellargraphs "ID"s are in the Neo4j graph
+        # use id() instead
```

Comment:
This may not be appropriate: https://neo4j.com/docs/cypher-manual/current/clauses/match/#match-node-by-id

> Neo4j reuses its internal ids when nodes and relationships are deleted. This means that applications using, and relying on internal Neo4j ids, are brittle or at risk of making mistakes. It is therefore recommended to rather use application-generated ids.

Reply:
ah good catch, i guess not everything is better with ilocs